### DISCUSSION TITLE
Sunday start date of the week

### DISCUSSION THREAD

**Discussion by lawrenciumchang** [kudrykv/latex-yearly-planner#83] [General]
Hello! Is there any way to have a planner generated with Sunday as the start date of the week?

**Comment by kudrykv** [comment]
Hello,

Look for ones that are named "Sunday first" https://github.com/kudrykv/latex-yearly-planner/discussions/9

**Reply by lawrenciumchang** [reply]
Ah thank you! I missed that somehow!
